.github/workflows/build-interop-docker.yml

```diff
@@ -1,11 +1,17 @@
 name: Build interop Docker image
 on: 
   push:
-    branches: [ interop ]
+    branches:
+      - master
+      - build-interop-docker-image-for-tags # TODO: remove
+    tags:
+      - 'v*'
```

Comment:
How are the tags for this repo created? I'm asking because if they're created automatically with the default GitHub Actions token then this trigger won't work unfortunately. There are some workarounds for that though.

Reply:
Currently I'm pushing them manually, but I was considering adopting the uCI release workflow.
How would those workaround work?

Reply:
With uCI release workflows, we'd need to add a `workflow_run` trigger (for successful release workflow runs) to this job and then use https://github.com/pl-strflt/uci/tree/main/.github/actions/inspect-releaser to retrieve information about the release that the release workflow possibly created. You can ping me when you want to set it up, happy to help :)